pkg/kubectl/apply/parse/util.go

```diff
@@ -101,13 +101,21 @@ func getFieldMeta(s proto.Schema, name string) (apply.FieldMetaImpl, error) {
 	m := apply.FieldMetaImpl{}
 	if s != nil {
 		ext := s.GetExtensions()
-		if s, found := ext["x-kubernetes-patch-strategy"]; found {
-			strategy, ok := s.(string)
+		if e, found := ext["x-kubernetes-patch-strategy"]; found {
+			strategy, ok := e.(string)
 			if !ok {
 				return apply.FieldMetaImpl{}, fmt.Errorf("Expected string for x-kubernetes-patch-strategy by got %T", s)
 			}
-			// TODO: Support multi-strategy
-			m.MergeType = strategy
+			if strings.Contains(strategy, ",") {
```

Comment:
Would it be simpler to remove that `if` statement, since `m.MergeType` is always equal to `strings.Split(strategy, ",")`, even if there is no comma?

Reply:
done